### PR TITLE
draft 2015-09-21-six-lines-to-install-and-start-sparkr-on-mac-os-x-yosemite

### PR DESCRIPTION
**Pull request by yukiegosapporo**

### CODE REVIEW

File: _posts/2015-09-21-six-lines-to-install-and-start-sparkr-on-mac-os-x-yosemite.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
+---
+layout: post
+title: Six lines to install and start SparkR on Mac OS X Yosemite
+excerpt: for both  and command line
```
Comment (Quiri) [inline _posts/2015-09-21-six-lines-to-install-and-start-sparkr-on-mac-os-x-yosemite.md]: its missing something... for both ? and command line

File: _posts/2015-09-21-six-lines-to-install-and-start-sparkr-on-mac-os-x-yosemite.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+categories: articles
+tags: [R, Spark, SparkR, Hadoop, big data]
+comments: true
+share: true
+published: true
```
Comment (Quiri) [inline _posts/2015-09-21-six-lines-to-install-and-start-sparkr-on-mac-os-x-yosemite.md]: can you change it to false, so I can preview it on locally before we publish it...

File: _posts/2015-09-21-six-lines-to-install-and-start-sparkr-on-mac-os-x-yosemite.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+{% highlight sh %}
+brew update # If you don't have homebrew, get it from here (http://brew.sh/)
+brew install hadoop # Install Hadoop
+brew install apache-spark # Install Spark
+{% endhighlight %}
+
+You can already start SparkR shell by typing this in your command line;
+
+```
+SparkR
```
Comment (Quiri) [inline _posts/2015-09-21-six-lines-to-install-and-start-sparkr-on-mac-os-x-yosemite.md:36]: maybe inline should be enough...

File: _posts/2015-09-21-six-lines-to-install-and-start-sparkr-on-mac-os-x-yosemite.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+
+> Apache Spark is a fast and general-purpose cluster computing system
+
+
+> SparkR is an R package that provides a light-weight frontend to use Apache Spark from R
+
+# Six lines to start SparkR
+The first three lines should be called in your command line.
+{% highlight sh %}
+brew update # If you don't have homebrew, get it from here (http://brew.sh/)
```
Comment (Quiri) [inline _posts/2015-09-21-six-lines-to-install-and-start-sparkr-on-mac-os-x-yosemite.md:28]: should also work with apt-get I assume

File: _posts/2015-09-21-six-lines-to-install-and-start-sparkr-on-mac-os-x-yosemite.md (hunk 3 above)
Comment (yukiegosapporo) [inline _posts/2015-09-21-six-lines-to-install-and-start-sparkr-on-mac-os-x-yosemite.md:36]: sh highlighter is better in comment highlighting

File: _posts/2015-09-21-six-lines-to-install-and-start-sparkr-on-mac-os-x-yosemite.md (hunk 1 above)
Comment (yukiegosapporo) [inline _posts/2015-09-21-six-lines-to-install-and-start-sparkr-on-mac-os-x-yosemite.md]: "RStudio" missing, will fix this

File: _posts/2015-09-21-six-lines-to-install-and-start-sparkr-on-mac-os-x-yosemite.md (hunk 2 above)
Comment (yukiegosapporo) [inline _posts/2015-09-21-six-lines-to-install-and-start-sparkr-on-mac-os-x-yosemite.md]: Yes

File: _posts/2015-09-21-six-lines-to-install-and-start-sparkr-on-mac-os-x-yosemite.md (hunk 4 above)
Comment (yukiegosapporo) [inline _posts/2015-09-21-six-lines-to-install-and-start-sparkr-on-mac-os-x-yosemite.md:28]: homebrew is the de facto mac package manager.
i didnt test apt-get so not this time.

File: _posts/2015-09-21-six-lines-to-install-and-start-sparkr-on-mac-os-x-yosemite.md (hunk 3 above)
Comment (Quiri) [inline _posts/2015-09-21-six-lines-to-install-and-start-sparkr-on-mac-os-x-yosemite.md:36]: then add {% highlight sh %} to SparkR

**Pull request merged**